tests/bson/bson-toPHP-008.phpt

```diff
@@ -0,0 +1,67 @@
+--TEST--
+MongoDB\BSON\toPHP(): Setting fieldPath typemaps for compound types with string keys
+--FILE--
+<?php
+
+require_once __DIR__ . "/../utils/basic.inc";
```

Comment:
`basic.inc` requires `PHONGO-SERVERS.json`. You should be use `require_once __DIR__ . '/../utils/tools.php';` for a non-server test.

Note: this wasn't a problem in [`bson-toPHP-007.phpt`](https://github.com/mongodb/mongo-php-driver/blob/master/tests/bson/bson-toPHP-007.phpt) from your earlier PR, as you didn't include anything there. That said, I don't think anything in this test actually needs a helper function in `tools.php`, since you're calling the BSON functions by their full namespace.